retrofit/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm'
+    id 'maven-publish'
+}
+
+group = reaktive_group_id
+version = reaktive_version
+
+publishing {
+    publications {
+        jvm(MavenPublication) {
+            from components.java
+        }
+    }
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation project(':reaktive')
+    implementation project(':reaktive-testing')
```

Comment:
`testImplementation`? What is the reason to bundle testing utils with main package?